src/ansiblelint/schemas/changelog.json

```
@@ -57,6 +123,100 @@
           },
           "type": "array"
         },
+        "objects": {
+          "additionalProperties": false,
+          "properties": {
+            "playbook": {
+              "items": {
+                "properties": {
+                  "description": {
+                    "markdownDescription": "Value of `short_description` from plugin `DOCUMENTATION`.",
```

Playbooks are not documentable yet.
```suggestion
                    "markdownDescription": "A short description of what the playbook does.",
```